### DISCUSSION TITLE
I want use ice storm , but i do not know how

### DISCUSSION THREAD

**Discussion by vvbrain** [icerpc/icerpc-csharp#4086] [General]
I reviewed the doc times by times, still do not have a clue.
How can I start the storm service?
On windows, with c#.Tools that downloaded never worked.May be the doc can have more details in the future,  or make the tool easy to use.

**Comment by pepone** [comment]
Did you check the IceStorm demos, I would start by looking at

https://github.com/zeroc-ice/ice-demos/tree/3.7/cpp98/IceStorm/clock

Once you got this demo working, you can also review the docs at 

https://doc.zeroc.com/ice/3.7/ice-services/icestorm/configuring-icestorm

**Comment by pepone** [comment]
@playg8 IceStorm is part of the [zeroc-ice/ice](https://github.com/zeroc-ice/ice) project. If you have further questions, feel free to open a discussion in the repository linked above.

**Reply by vvbrain** [reply]
![image](https://github.com/user-attachments/assets/b8e59755-7248-4ab4-8e69-a36434e72b9e)
I wrote two config files(config.icebox,config.service).Once I run iceboxnet.exe, I got error in command line.
`C:\Users\ljl\Desktop\IceTools\tools\net45>iceboxnet
未经处理的异常:  System.IO.FileNotFoundException: 未能加载文件或程序集“Ice, Version=3.7.10.0, Culture=neutral, PublicKeyToken=0c5ebb72d74932c6”或它的某一个依赖项。系统找不到指定的文件。
   在 IceBox.Server.Main(String[] args)`
Since I am new to zeroc-ice, I do not know what happend. May be I missed something importent, but I can not findout what was it. Need help really.

**Reply by pepone** [reply]
Hi @playg8,

Can you open a discussion for this issue in https://github.com/zeroc-ice/ice repository, IceStorm is not part of IceRPC. This is the repository for IceRPC for C# which is unrelated to IceStorm.

Cheers,
Jose

**Reply by vvbrain** [reply]
The tool under D:\Program Files\ZeroC\Ice-3.7.10\bin maybe work, when I tried it under the folder,no error occured. This is a big step for me.

**Reply by vvbrain** [reply]
May be this method of start the storm can be written in doc, can be a lot of help for many of users.

**Comment by vvbrain** [comment]
Jus wait, I will try the cmd under D:\Program Files\ZeroC\Ice-3.7.10\bin I installed.

**Reply by pepone** [reply]
I deleted your other issue. As I wrote about IceStorm is not related to this repository. Please if you have any further question regarding IceStorm open an discussion in the https://github.com/zeroc-ice/ice repository.

locking this issue as it is unrelated to IceRPC.
